inc/template-tags.php

```diff
@@ -227,19 +227,16 @@ function twenty_twenty_one_the_posts_navigation() {
 			array(
 				'mid_size'  => 2,
 				'prev_text' => sprintf(
-					/* Translators: Left arrow */
 					'%s <span class="nav-prev-text">%s</span>',
```

Comment:
Good catch on the link text!

Why was the translators comment removed?

Reply:
Because it doesn't actually apply to anything here— the placeholder is not translated, only the text is.